filters-share/specific_ELEMHIDE.txt

```diff
@@ -59,6 +59,9 @@ sports.donga.com##div[id*="_AD"]
 ! ##div[id$="_Ad"]
 yes24.com##div[id$="_Ad"]
 !
+hogaeng.co.kr###partnership
+hogaeng.co.kr##.sponsor
+hogaeng.co.kr##.sponsor2
```

Comment:
Can be compressed.

Reply:
like hogaeng.co.kr##div[id^="sponsor"] ?

Reply:
Yes.

Reply:
resloved pls re review